### ISSUE TITLE
Breadcrumbs in sub-modules should link to pages, not directories.

### ISSUE THREAD

**Issue by christian-schlichtherle** [andriusvelykis/reflow-maven-skin#28]
... because otherwise the links are broken when not using a web server, e.g. when issuing

```
$ mvn site site:stage
$ open target/staging/index.html
```

and then browsing to any submodule.

**Comment by andriusvelykis** [comment]
Yes, browsing the submodules locally is an issue when the links are to directories. However, on the web I personally prefer the directory links rather than `index.html`.

Anyway, what is the default for other skins? Do they link to `index.html` pages? Because I thought I did not change how the breadcrumb links are generated..

Also, you can customise what/how the breadcrumb links are rendered using the `<breadcrumbs>` element in your `site.xml`, e.g.:

```
<breadcrumbs>
  <item name="Reflow" href="/index.html" />
</breadcrumbs>
```

**Comment by christian-schlichtherle** [comment]
Hi Andrius,

thanks for looking into this. Yes, this works in other skins and is especially important with multi-module projects where you typically do not want to customize breadcrumbs in each site.xml. Since the whole point of the doxia framework is to create a static web site, it should really work without a web server, so any skin should always link to individual pages, not directories.

I will get back to the other issues later.

Regards,
Christian

Am 29.01.2014 um 12:49 schrieb Andrius Velykis notifications@github.com:
